bin/update-yarn.sh

```diff
@@ -21,7 +21,9 @@ PLUGINS=$(awk '{ if ($1 == "spec:") print $2 }' .yarnrc.yml)
 echo "$PLUGINS" | xargs -n1 -t corepack yarn plugin remove
 
 cp package.json .yarn/cache/tmp.package.json
-sed "s#yarn\": \"$CURRENT_VERSION\"#yarn\": \"$LAST_VERSION\"#;s#\"yarn@$CURRENT_VERSION\"#\"yarn@$LAST_VERSION\"#" .yarn/cache/tmp.package.json > package.json
+sed "s#\"yarn\": \"$CURRENT_VERSION\"#\"yarn\": \"$LAST_VERSION\"#;s#\"yarn@$CURRENT_VERSION\"#\"yarn@$LAST_VERSION\"#" .yarn/cache/tmp.package.json > package.json
```

Comment:
I think it can improved by using `sed -i` (in-place edit), then no need for these temp files?

Reply:
`-i` is not POSIX (it's GNU-only), it doesn't work everywhere. Maybe it's better to keep a portable version (I'd prefer not to have to boot up a VM when I want to test the commands from macOS).

Reply:
ok I agree